### PR TITLE
Issue #122 Adding Cache Purges Button to Navigation Drawer

### PR DESCRIPTION
**Pull request by mike44441**
Created a Cache Purges Button to Navigation Sidebar with unique icon.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,5 @@ node_modules
 .env
 /styleguide
 .coverage
+src/views/.DS_Store
```
Comment (helfi92) [inline .gitignore]: Replace with `.DS_Store`, instead of `src/views/.DS_Store`.

File: .gitignore [hunk 2]
```diff
@@ -7,3 +7,5 @@ node_modules
...
+config.env
```
Comment (helfi92) [inline .gitignore]: What is this for?

Comment (mike44441) [reply .gitignore]: Ah that was an extra file I created. I'll remove everything that isn't the main modified file then.

**Comment by mike44441** [comment]
I reverted my previous change and had the commit not change the .DS_Store and .gitignore files.

File: .gitignore [hunk 3]
```diff
@@ -7,3 +7,4 @@ node_modules
 .env
 /styleguide
 .coverage
+.DS_Store
```
Comment (helfi92) [inline .gitignore]: I can still see `src/.DS_Store`, `src/components/.DS_Store` and `src/components/Dashboard/.DS_Store` commited:

screen shot 2018-09-27 at 8 22 55 am

You'll have to do something like:

```bash
git rm src/.DS_Store
git rm src/components/.DS_Store
git rm src/components/Dashboard/.DS_Store
git commit -m "remove DS_Store"
```

Then the last step would be to `git push` to this branch.

Comment (mike44441) [reply .gitignore]: Thanks for the help and I have just done finished the push. =)

**Comment by mike44441** [comment]
I have pushed the changes, let me know if there is anything else!

**Pull request merged**